src/hoverboard-app.html

```diff
@@ -628,6 +674,31 @@ <h3 class="location">{$ location.short $}</h3>
       _removeQueryParams() {
         this.set('queryParams', null);
       }
+
+      _toggleNotifications() {
+        this._closeNotificationMenu();
+        if (this.notifications.status === NOTIFICATIONS_STATUS.GRANTED) {
+          notificationsActions.unsubscribe(this.notifications.token);
```

Comment:
why you unsubscribe when permission is granted?

Reply:
if people get rid of notifications and prevent them blocking notifications on the website